apps/ios_rpc/tvmrpc/TVMRuntime.mm

```diff
@@ -71,88 +73,19 @@ void LogMessageImpl(const std::string& file, int lineno, const std::string& mess
 namespace tvm {
 namespace runtime {
 
-class NSStreamChannel final : public RPCChannel {
```

Comment:
just curious why this is going away? not needed if we use standard sockets?

Reply:
Moved into "RPCServer.mm". And now it's wrapped into `PackedFunction` to meet API requirements of `rpc.CreateEventDrivenServer`.